// Proto file for connection between the client and intermediate balancer
// python3 -m grpc_tools.protoc -I. --python_out=. --pyi_out=. --grpc_python_out=. trial_2.proto

syntax = "proto3";
package trial_2;

// To store the message fromm the client
message FunctionMessage
{
    optional int32 data1 = 1;
    optional int32 data2 = 2;
    optional int32 function = 3;
}

message ReturnValue
{
    optional float val = 1;
}

// Void message for when nothing actually needs to be returned
message void {}

service Alert
{
    rpc RelayClientMessage(FunctionMessage) returns (ReturnValue) {}
}